mdpow/forcefields.py

```diff
@@ -192,6 +194,8 @@ def get_water_model(watermodel=DEFAULT_WATER_MODEL):
         identifier="wetoctanol", itp="1octwet.itp", coordinates="1octwet_amber.gro"),
     'cyclohexane': GromacsSolventModel(
         identifier="cyclohexane", itp="1cyclo.itp", coordinates="1cyclo_amber.gro"),
+    'toluene': GromacsSolventModel(
+        identifier="toluene", itp="1tol.itp", coordinates="1tol_gaff.gro"),
```

Comment:
I see that for the other solvents we used "amber" instead of "gaff", so the file should be renamed for consistency.